src/meta/ClusterInfo.cpp

```diff
@@ -256,14 +256,11 @@ unordered_map<std::string, Column> asColumnProps(const YAML::Node& node) {
   return props;
 }
 
-KafkaSerde asSerde(const YAML::Node& node) {
+KafkaSerde asSerde(const YAML::Node& node, const double retention) {
   KafkaSerde serde;
   if (node) {
     serde.protocol = node["protocol"].as<std::string>();
-    auto retention = node["retention"];
-    if (retention) {
-      serde.retention = retention.as<uint64_t>();
-    }
+    serde.retention = (uint64_t) retention;
```

Comment:
This retention is KAFKA topic retention in Kafka store, not the max-hr (retention in Nebula), the usage of this value is used to estimate offset and range for each Kafka split when early version of Kafka doesn't have watermark/offset feature, we should leave this as it is.

Reply:
got it, how about rename to topic-retention to avoid confusion.